.github/workflows/branch-environment.yml

```
@@ -0,0 +1,286 @@
+    name: Wait for Supabase Branch
+    needs: detect-platform
+    if: github.event.action != 'closed'
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    outputs:
+      supabase-url: ${{ steps.wait.outputs.supabase-url }}
+      supabase-ready: ${{ steps.wait.outputs.ready }}
+    steps:
+      - name: Wait for Supabase branch status
+        id: wait
+        run: |
```

actionlint (1.7.9)

"github.event.pull_request.head.ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)